src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbsListBody.tsx

```diff
@@ -26,20 +30,37 @@ const BreadcrumbsListBody = ({
     {collapsedQuantity > 0 && (
       <BreadcrumbCollapsed onClick={onToggleCollapse} quantity={collapsedQuantity} />
     )}
-    {breadcrumbs.map(({color, borderColor, icon, ...crumb}, idx) => {
-      const hasError =
-        crumb.type === BreadcrumbType.MESSAGE || crumb.type === BreadcrumbType.EXCEPTION;
+    {breadcrumbs.map(({color, icon, ...crumb}, idx) => {
+      const hasError = crumb.type === BreadcrumbType.ERROR;
       return (
-        <BreadcrumbListItem key={idx} data-test-id="breadcrumb" hasError={hasError}>
-          <BreadcrumbIcon icon={icon} color={color} borderColor={borderColor} />
-          <BreadcrumbCategory category={crumb.category} />
-          <BreadcrumbData breadcrumb={crumb as Breadcrumb} />
-          <BreadcrumbLevel level={crumb.level} />
-          <BreadcrumbTime timestamp={crumb.timestamp} />
-        </BreadcrumbListItem>
+        <React.Fragment key={idx}>
+          <GridCell hasError={hasError} withoutBorder={['right']} withBeforeContent>
```

Comment:
Since we always have and know which two cells have borders, I would just do something like:

```
const LeftGridCell = styled(GridCell)` border-left: ...`
```

There's not much reason to have it be a prop for the styled component

Reply:
I think similarly, `withBeforeContent` may not need to be a prop either, it's always on the left cell, no?

(`withBeforeContent` is also not a great name because it's just describing the implementation vs. what it actually is -- also I have no idea what it's called either).